### PR TITLE
Cleanup dev

### PR DESCRIPTION
**Pull request by laborleben**
Reducing chunk size to 1000000 (1M) in general and cleaning up and removing trailing white spaces.

### CODE REVIEW

File: testbeam_analysis/dut_alignment.py [hunk 1]
```diff
@@ -708,7 +708,7 @@ def apply_alignment(input_hit_file, input_alignment_file, output_hit_file, inver
         Filename of the input alignment file.
     output_hit_file : string
         Filename of the output hits file with hit data after alignment was applied.
-    inverse : boolean
+    inverse : bool
```
Comment (DavidLP) [inline testbeam_analysis/dut_alignment.py:711]: This introduces a regression here: https://github.com/SiLab-Bonn/testbeam_analysis/blob/99f8ab255e2207436cac56bab3770550f1178ec8/testbeam_analysis/gui/analysis_widgets.py#L328

File: testbeam_analysis/result_analysis.py [hunk 2]
```diff
@@ -17,7 +17,7 @@
 from testbeam_analysis.tools import analysis_utils
 
 
-def calculate_residuals(input_tracks_file, input_alignment_file, n_pixels, pixel_size, output_residuals_file=None, dut_names=None, use_duts=None, max_chi2=None, nbins_per_pixel=None, npixels_per_bin=None, force_prealignment=False, use_fit_limits=True, plot=True, cluster_size_selection=None, gui=False, chunk_size=1000000):
+def calculate_residuals(input_tracks_file, input_alignment_file, n_pixels, pixel_size, output_residuals_file=None, dut_names=None, use_duts=None, max_chi2=None, nbins_per_pixel=None, npixels_per_bin=None, force_prealignment=False, use_fit_limits=True, cluster_size_selection=None, plot=True, gui=False, chunk_size=1000000):
```
Comment (DavidLP) [inline testbeam_analysis/result_analysis.py:20]: Yes, better order

File: testbeam_analysis/tools/smc.py [hunk 3]
```diff
@@ -297,7 +297,7 @@ def _combine(self):
         else:  # TODO: solution without having all hists in RAM
             # Only one file, merging not needed
             if len(self.tmp_files) == 1:
-                shutil.move(self.tmp_files[0], self.file_out)   
+                shutil.move(self.tmp_files[0], self.file_out)
```
Comment (DavidLP) [inline testbeam_analysis/tools/smc.py:300]: I really wonder where the white spaces come from

**Review by DavidLP — CHANGES_REQUESTED** [review]
Look fine besides one change that is needed in the type inference function of the GUI.

File: testbeam_analysis/dut_alignment.py (hunk 1 above)
Comment (laborleben) [reply testbeam_analysis/dut_alignment.py:711]: This is more of an issue for the gui branch than development?

Comment (laborleben) [reply testbeam_analysis/dut_alignment.py:711]: but ```bool``` is the type name and ```boolean``` should be avoided?

**Review by DavidLP — APPROVED** [review]

**Pull request merged**